packages/oscal-react-library/src/components/StyledAccordion.tsx

```diff
@@ -0,0 +1,12 @@
+import ArrowForwardSharp from "@mui/icons-material/ArrowForwardIosSharp";
+import MuiAccordion from "@mui/material/Accordion";
+import MuiAccordionSummary from "@mui/material/AccordionSummary";
+import React from "react";
+
+export const Accordion = (props: any) => (
+  <MuiAccordion className="Accordion" disableGutters elevation={0} square {...props} />
+);
+
+export const AccordionSummary = (props: any) => (
+  <MuiAccordionSummary className="AccordionSummary" expandIcon={<ArrowForwardSharp />} {...props} />
+);
```

Comment:
Receiving an error when attempting to add `as typeof MuiAccordion`:

```
Conversion of type 'Element' to type 'OverridableComponent<AccordionTypeMap<{}, "div">>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)
```

I don't think we need to include the `typeof` for this to be valid TypeScript. Is providing the type for `props` not enough?